src/structs.py

```diff
@@ -841,6 +842,7 @@ def add(self, member: Tuple[Segment, ObjToVarSub]) -> None:
 # Convenience higher-order types useful throughout the code
 StateActionTrajectory = Tuple[List[State], List[Action]]
 OptionTrajectory = Tuple[List[State], List[_Option]]
+OptionSpec = Tuple[ParameterizedOption, List[Variable]]
```

Comment:
no action required, but I just want to remark that this is basically an "action template" per definition 1 of LOFT paper https://arxiv.org/pdf/2103.00589.pdf